p2p/libp2p_bridge/test_grpc.py

```diff
@@ -23,32 +23,16 @@
 import p2p.libp2p_bridge.github.com.ethresearch.sharding_p2p_poc.pb.rpc.rpc_pb2_grpc as rpc_pb2_grpc
 
 
+def make_collation_topic(shard_id):
+    return COLLATION_TOPIC_FORMAT.format(shard_id)
+
+
 def make_event_stub():
     dial_addr = "{}:{}".format(RPC_SERVER_LISTEN_IP, RPC_SERVER_PORT)
     channel = grpc.insecure_channel(dial_addr)
     return event_pb2_grpc.EventStub(channel)
 
 
-def send_receive():
-    """Test if `Receive` servicer works
-    """
-    stub = make_event_stub()
-    cr = CollationRequest(1, 2, "")
-    req = event_pb2.ReceiveRequest(
-        peerID="",
-        topic="",
-        msgType=MsgType.CollationRequest,
-        data=cr.to_bytes(),
-    )
-    stub.Receive(req)
```

Comment:
Why did this test get removed?

Reply:
Ah actually this function is not called previously, so I removed it. But you're right this test currently lacks of the tests for `GRPCServer` using `make_event_stub`. Will Add more tests for this.